spec/interpret_date_spec.rb

```diff
@@ -74,4 +74,32 @@
       end
     end
   end
+
+  describe "#interpret_date_assignment_for" do
```

Comment:
Class methods are annotated with a `.` instead of the `#` used for instance methods.